### PR TITLE
fix/push-empty-files

### PR DESCRIPTION
**Pull request by AcckiyGerman**
Error message added for empty tabular files
https://github.com/datahq/data-cli/issues/257
https://github.com/datahq/datahub-qa/issues/60

### CODE REVIEW

File: bin/data-push.js [hunk 1]
```diff
@@ -146,7 +146,16 @@ const prepareDatasetFromFile = async filePath => {
   // List of formats that are known as tabular
   const knownTabularFormats = ['csv', 'tsv', 'dsv']
   if (knownTabularFormats.includes(file.descriptor.format)) {
-    await file.addSchema()
+    try {
+      await file.addSchema()
+    } catch(err){
+      error("tabular file is invalid: " + file.path)
```
Comment (anuveyatsu) [inline bin/data-push.js:152]: What exactly can be invalid here? I guess `file.addSchema` throws error when tabular data is empty, right? What other situations?

**Comment by AcckiyGerman** [comment]
I don't know... No headers?

File: bin/data-push.js (hunk 1 above)
Comment (AcckiyGerman) [reply bin/data-push.js:152]:
```
$ data push this_is_excel_file.csv --debug
> Error! Invalid opening quote at line 1
```

Comment (AcckiyGerman) [reply bin/data-push.js:152]: I could imaging that `addSchema()` could fail for many reasons
- file is not readable   (no this one will cause exception in `data.load()`)
- this is actually not `csv`
- unknown separator ?

Comment (anuveyatsu) [reply bin/data-push.js:152]: @AcckiyGerman 
- `addSchema` would fail for non-tabular data
- it handles separators, if one of `, ; | ^ : * &`

**Comment by anuveyatsu** [comment]
@AcckiyGerman ok, I think it is sufficient so let's merge this one

File: bin/data-push.js (hunk 1 above)
Comment (AcckiyGerman) [reply bin/data-push.js:152]: So do you want me to change the error message or leave it as is? - aha, see, we merged already

**Pull request merged**